File: tests/fixtures/csharp/broken_syntax.cs
// Deliberately broken C# — used to verify parse-error counting.
// tree-sitter-c-sharp surfaces ERROR / MISSING nodes for missing
// braces, broken method signatures, and incomplete expressions.

namespace Demo.Broken
{
    public class Ok
    {
        public int Good() { return 1; }
    }

    public class Busted
    {
        public void Broken(int x {
            int y = ;
        }

        public int StillVisible => 42;

File: tests/fixtures/java/broken_syntax.java
package demo;

// Deliberately broken — used to verify parse-error counting and the
// header warning. Missing close brace on `broken(`, missing expression
// before `;`, unclosed class body. tree-sitter should still produce a
// partial tree with ERROR / MISSING nodes.
public class Broken {
    public int ok() { return 1; }

    public void broken( {
        int x = ;
    }

    public int stillVisible() { return 2; }

